app/assets/config/manifest.js

```diff
@@ -0,0 +1,13 @@
+//= link accessible-autocomplete/dist/accessible-autocomplete.min.css
+//= link application.css
+//= link print.css
+//= link joint.css
+//= link visualise.css
+//= link accessible-autocomplete/dist/accessible-autocomplete.min.js
+//= link application.js
+//= link dagre.js
+//= link joint.layout.DirectedGraph.js
+//= link joint.js
+//= link joint.patch.js
+//= link test-dependencies.js
+//= link visualise.js
```

Comment:
I imagine you can simplify this a little:

```
//= link_directory ./stylesheets
//= link joint.css
//= link application.js
//= link dagre.js
//= link joint.layout.DirectedGraph.js
//= link joint.js
//= link joint.patch.js
//= link test-dependencies.js
//= link visualise.js
```

You should also remove the assets_precompile stuff from https://github.com/alphagov/smart-answers/blob/d3fc1b1ff37a9163cdbb020e2e2fd7c52d4f77ea/config/initializers/assets.rb#L14-L18 and https://github.com/alphagov/smart-answers/blob/d3fc1b1ff37a9163cdbb020e2e2fd7c52d4f77ea/config/application.rb#L63-L72

A couple of suggestions that could make this part of the update a little easier are:

- Do the sprockets update as part of the dependabot PR: https://github.com/alphagov/smart-answers/pull/5879 (which updates Sprockets) so you can separate it out from the Rails 7 stuff
- - Perhaps use asset-pipeline to require the various joint.* js files from visualise.js, I don't think they're used anywhere else.